doc/api/tls.md

```diff
@@ -876,6 +876,30 @@ socket.on('end', () => {
 });
 ```
 
+Passing in `net.Socket` will start a new instance.
+To upgrade an existing instance of `net.Socket` to a 
```

Comment:
Please try to avoid trailing whitespace characters. Git can help you with that, see [this](https://stackoverflow.com/questions/591923/).

(This is not critical, our team will usually fix this while landing the PR, but it is still good practice to take care of that! :smiley: )